lib/rules/rules.json

```diff
@@ -1,37 +1,43 @@
 {
-  "concat": "Consider using the native Array.prototype.concat()",
-  "fill": "Consider using the native ES6 Array.prototype.fill()",
-  "find": "Consider using the native ES6 Array.prototype.find()",
-  "detect": "Consider using the native ES6 Array.prototype.find()",
-  "findIndex": "Consider using the native ES6 Array.prototype.findIndex()",
-  "indexOf": "Consider using the native Array.prototype.indexOf()",
-  "join": "Consider using the native Array.prototype.join()",
-  "lastIndexOf": "Consider using the native Array.prototype.lastIndexOf()",
-  "reverse": "Consider using the native Array.prototype.reverse()",
-  "each": "Consider using the native Array.prototype.forEach()",
-  "forEach": "Consider using the native Array.prototype.forEach()",
-  "every": "Consider using the native Array.prototype.every()",
-  "all": "Consider using the native Array.prototype.every()",
-  "filter": "Consider using the native Array.prototype.filter()",
-  "select": "Consider using the native Array.prototype.filter()",
-  "includes": "Consider using the native Array.prototype.includes()",
-  "contains": "Consider using the native Array.prototype.includes()",
-  "map": "Consider using the native Array.prototype.map()",
-  "collect": "Consider using the native Array.prototype.map()",
-  "reduce": "Consider using the native Array.prototype.reduce()",
-  "inject": "Consider using the native Array.prototype.reduce()",
-  "foldl": "Consider using the native Array.prototype.reduce()",
-  "reduceRight": "Consider using the native Array.prototype.reduceRight()",
-  "foldr": "Consider using the native Array.prototype.reduceRight()",
-  "size": "Consider using the native Array.prototype.length",
-  "some": "Consider using the native Array.prototype.some()",
-  "any": "Consider using the native Array.prototype.some()",
-  "isNaN": "Consider using the native isNaN() or ES6 Number.isNaN()",
-  "extendOwn": "Consider using the native ES6 Object.assign()",
-  "assign": "Consider using the native ES6 Object.assign()",
-  "keys": "Consider using the native Object.keys()",
-  "repeat": "Consider using the native ES6 String.prototype.repeat()",
-  "toLower": "Consider using the native String.prototype.toLowerCase()",
-  "toUpper": "Consider using the native String.prototype.toUpperCase()",
-  "trim": "Consider using the native String.prototype.trim()"
+  "concat":      { "compatible": true, "alternative": "Array.prototype.concat()" },
```

Comment:
We can add more metadata to this file, such as which browsers are supported. Eventually we may even generate the documentation and run automated compatibility tests from the configuration file.